_includes/main.liquid

```diff
@@ -69,6 +69,22 @@
                 </span>
             </div>
         </div>
+        <div class="notification" key="smartcon-2021" start="2021-07-09" end="2021-08-04">
+            <div class="notification-info">
+                <span class="notification-close">X</span>
+                <span class="notification-text">
+                    SmartCon is back August 5-7. <a href="https://hopin.com/events/smart-contract-summit-1?ref=252e0a3dfb0a">Join us</a> for 3 days of workshops, panels, AMAs, and more.
+                </span>
+            </div>
+        </div>
+        <div class="notification" key="smartcon-live-2021" start="2021-08-04" end="2021-08-08">
+            <div class="notification-info">
+                <span class="notification-close">X</span>
+                <span class="notification-text">
+                    SmartCon 2021 is up. Come <a href="https://www.smartcontractsummit.io/">watch live</a>!
```

Comment:
Can we tweak this text to say "SmartCon is happening right now. Come watch live!"